client/pages/fixtures.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import formatTime from './format-time';
+const Fixtures = props => {
```

Comment:
I might be confused about what "fixtures" means, but doesn't this component render _one_ fixture?